### PR TITLE
Adds a rad_tick readout to bodyscans

### PR DESCRIPTION
**Pull request by Pathid**
Now you can really know how fucked somebody is from the radiation system. I labeled the number "Absorbed rads danger index" instead of rad_tick but that's open to change.
:cl:
* rscadd: Body scans now show a number for the progress of radiation poisoning, in addition to the radiation level, which doesn't tell the whole story.

### CODE REVIEW

File: code/game/machinery/adv_med.dm [hunk 1]
```diff
@@ -371,6 +372,8 @@
 	dat += text("[]\t-Burn Severity %: []</font><br><br>", (occ["fireloss"] < 60 ? "<font color='blue'>" : "<font color='red'>"), occ["fireloss"])
 
 	dat += text("[]\tRadiation Level %: []</font><br>", (occ["rads"] < 10 ?"<font color='blue'>" : "<font color='red'>"), occ["rads"])
+	if(occ["radtick"] > 0)
+		dat += text("<font color='red'>Absorbed rads danger index: <b>[occ["radtick"]]</b></font><br>")
```
Comment (jknpj) [inline code/game/machinery/adv_med.dm:376]: i know rads is rad but why not 'radiation'

File: code/modules/mob/living/simple_animal/borer/verbs_attached_chest.dm [hunk 2]
```diff
@@ -122,6 +123,8 @@
 	if(occ["virus_present"])
 		dat += "<font color='red'>Viral pathogen detected in blood stream.</font><br>"
 	dat += text("[]\tRadiation Level %: []</font><br>", (occ["rads"] < 10 ?"<font color='blue'>" : "<font color='red'>"), occ["rads"])
+	if(occ["radtick"] > 0)
+		dat += text("<font color='red'>Absorbed rads danger index: <b>[occ["radtick"]]</b></font><br>")
```
Comment (jknpj) [inline code/modules/mob/living/simple_animal/borer/verbs_attached_chest.dm:127]: ditto

File: code/game/machinery/adv_med.dm (hunk 1 above)
Comment (Pathid) [reply code/game/machinery/adv_med.dm:376]: Guess I was trying to shorten the name. Radiation is fine though.

Comment (Pathid) [reply code/game/machinery/adv_med.dm:376]: Although, you know, a rad is an actual measurement of absorbed radiation dose. Wasn't really thinking of that at the time but it's somewhat appropriate.

**Pull request merged**